.github/workflows/add-sponsors.yml

```
@@ -0,0 +1,31 @@
+    branches: [main, master, feature-develop, feat/develop]
+  schedule:
+    - cron: "30 15 * * 0-6" # Daily at 15:30 UTC
+
+permissions:
+  contents: write
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
```

actionlint (1.7.7)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)